src/components/base/SaunaImage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+// TODO : Sauna Bilder anpassen
+
+type Props ={
```

Comment:
I would export the props, so the type can be used somewhere else, if needed. And we should probably use the name in the component, so my suggestion:
```ts
export type SaunaImageProps = {
```

Reply:
Props have been renamed "SaunaImageProps" and exported

Reply:
1: I would move the SaunaImage to a folder like components/saunas or sth like this (we can discuss tomorrow in the meeting).

 -> SaunaImage changed to " SaunaCard " and moved to new folder "components/saunas"

2. We will probably have an entity Sauna in the future. Then it will make sense to pass a sauna via the props to the SaunaImage and the SaunaImage could then figure out most of the props on it's own (like name, description, link to the sauna, etc.
  
-> Props got replaced by new Type " Sauna.ts " 


-> Basic Tests for SaunaCard & Overview have been initialized. ( Sonar rated Code very bad after that , because testing code was declared as" untested" ..? any Idea ? )